src/catalog/catalog_entry/macro_catalog_entry.cpp

```diff
@@ -0,0 +1,29 @@
+#include "duckdb/catalog/catalog_entry/macro_catalog_entry.hpp"
+
+#include "duckdb/common/serializer.hpp"
+
+namespace duckdb {
+
+void MacroCatalogEntry::Serialize(Serializer &serializer) {
+	serializer.WriteString(schema->name);
+	serializer.WriteString(name);
+    function->expression->Serialize(serializer);
+    serializer.Write<uint32_t>((uint32_t)function->parameters.size());
+	for (auto &param : function->parameters) {
```

Comment:
There seems to be some weird formatting here, likely a mix of tabs and spaces. Could you run the formatter (`python3 scripts/format.py master`)?